api/server/server.go

```diff
@@ -1628,6 +1628,7 @@ func ServeApi(job *engine.Job) engine.Status {
 				chErrors <- err
 				return
 			}
+			defer removeUnixSocket(protoAddr)
```

Comment:
What's point of defer here? You can just write it after `chErrors <- srv.Serve()`